esptfa_arima/Test_Management/views.py

```
@@ -512,52 +674,78 @@ class ActualPostTestViewSet(viewsets.ModelViewSet):
-                # Delete existing actual scores for this doc if any? 
+                # Delete existing actual scores for this doc if any?
                 # Or just update? Let's update or create.
                 for score_item in scores:
-                    lrn = score_item.get('lrn')
-                    score = score_item.get('score')
+                    lrn = score_item.get("lrn")
+                    score = score_item.get("score")
                     max_score = document.post_test_max_score
-                    
+
                     student = Student.objects.get(lrn=lrn)
```

Ruff (0.14.14)

[error] `Student` may be undefined, or defined from star imports

(F405)